src/redux/books/BooksSlice.js

```diff
@@ -0,0 +1,30 @@
+// Actions
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+// Reducer
+export default function booksReducer(state = [], action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        action.payload.book,
+      ];
+
+    case REMOVE_BOOK:
+      return [...state.filter((book) => (book.id !== action.payload.id))];
+
+    default:
+      return state;
+  }
+}
+
+// Action Creators
+
+const addBookAction = (book) => ({ type: ADD_BOOK, payload: { book } });
+
+const removeBookAction = (id) => ({ type: REMOVE_BOOK, payload: { id } });
+
+export {
+  ADD_BOOK, REMOVE_BOOK, addBookAction, removeBookAction,
+};
```

Comment:
- [ ] Good job following the duck pattern in this file and its working but the issue is that you are required in this milestone to use the new and better way to write action creates and reducers which is through redux toolkit's feature `createSlice`. kindly consider writing the same functionality using createSlice feature of redux toolkit. you need to implement following the requirements below and you can also take help from redux toolkit official documentation.
- Define a slice of state for books that:
   -  Includes an array of books (initial state: empty array)
   -  Includes a reducer that adds a book
   -  Includes a reducer that removes a book